app/src/main/kotlin/com/wire/android/ui/home/HomeTopBar.kt

```diff
@@ -42,9 +42,11 @@ fun HomeTopBar(
     val scrollDownState = viewModel.scrollDownFlow.collectAsState(false)
     val firstLineElevation = if (!isSearchable || scrollDownState.value) dimensions().topBarElevationHeight else 0.dp
 
-    Box(
-        Modifier.background(Color.Transparent)
-    ) {
+    LaunchedEffect(viewModel) {
+        viewModel.loadUserAvatar()
+    }
+
+    Box {
```

Comment:
I changed a lot this HomeTopBar function in my PR so I guess I am going to refactor it anyway, but I think it would make more sense to just pass bytearray here and hoist loading it up to the parent